components/widgets/jenkins/index.js

```diff
@@ -0,0 +1,86 @@
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import styled from 'styled-components'
+import Widget from '../../widget'
+import Table, { Th, Td } from '../../table'
+import Badge from '../../badge'
+
+const JenkinsBadge = styled(Badge)`
+  background-color: ${props => {
+    switch (props.status) {
+      case 'FAILURE':
+        return props.theme.palette.errorColor
+      case 'UNSTABLE':
+        return props.theme.palette.warnColor
+      case 'SUCCESS':
+        return props.theme.palette.successColor
+      default:
+        return 'transparent'
+    }
+  }}
+`
+
+export default class Jenkins extends Component {
+  static defaultProps = {
+    jobs: [],
```

Comment:
if it is empty, it wouldn´t fetch any information from the api and with `builds && builds.map` it will not fail in the render method :)